src/serverenvironment.cpp

```diff
@@ -603,6 +603,17 @@ void ServerEnvironment::kickAllPlayers(AccessDeniedCode reason,
 		m_server->DenyAccess(player->getPeerId(), reason, str_reason, reconnect);
 }
 
+void ServerEnvironment::transferPlayer(const std::string &name, const std::string &address, const std::string &port)
+{
+	auto addressPack = address + ";" + port;
+
+	for (RemotePlayer *player : m_players) {
+		if (name.compare(player->getName()) == 0) {
```

Comment:
Isn't there some kind of "get player by name" function you could use?

I also think that this should ideally throw an error if no player with the given name is found (rather than failing silently).

Reply:
It will be probably good to return false as `l_disconnect_player` in `l_server.cpp` does.

Reply:
That depends on whether you would consider this a frequent, recoverable exception (that often does not need to be handled) or an "unrecoverable" "panic" / failure, IMO. I think passing a wrong player name is clearly a logic issue and hence of the latter kind, so should trigger an error. Modders can just check whether a player is online if they want the "lax" behavior (or they could use `pcall`).